package.json

```diff
@@ -12,7 +12,7 @@
   "repository": "https://github.com/atom/one-dark-ui",
   "main": "lib/main",
   "engines": {
-    "atom": ">0.40.0"
+    "atom": ">1.0.11"
```

Comment:
Ah yeah we totally do! https://github.com/atom/apm/blob/f18c91d2c0f41cf5d063e5127b322a51478cd50e/src/command.coffee#L63-L68

So this should be good to go